libyul/optimiser/DataFlowAnalyzer.cpp

```diff
@@ -218,6 +219,18 @@ void DataFlowAnalyzer::operator()(Block& _block)
 	assertThrow(numScopes == m_variableScopes.size(), OptimizerException, "");
 }
 
+optional<YulString> DataFlowAnalyzer::storageValue(YulString _key) const
+{
+	auto it = m_state.storage.find(_key);
+	return it == m_state.storage.end() ? nullopt : optional<YulString>{it->second};
```

Comment:
```suggestion
	if (auto const* value = util::valueOrNullptr(m_state.storage, _key))
		return *value;
	else
		return nullopt;
```
Or we could add something like ``util::valueOrNullopt`` or ``util::valueAsOptional`` or so... but well, also fine as is...